cpu/fe310/Kconfig

```diff
@@ -18,6 +18,10 @@ config CPU_FAM_FE310
     select HAS_PERIPH_RTT_SET_COUNTER
     select HAS_PERIPH_WDT
 
+    select MODULE_PERIPH_PLIC if TEST_KCONFIG
+    select MODULE_PERIPH_CORETIMER if MODULE_PERIPH_TIMER && HAS_PERIPH_CORETIMER
+    select MODULE_PERIPH_RTT if MODULE_PERIPH_RTC && HAS_PERIPH_RTT
```

Comment:
Seem like `ztimer_periph_rtc` is not the default backend in `Makefile.dep` you could either force it in the application through Kconfig or add the following

```

ifneq (,$(filter ztimer_sec,$(USEMODULE)))
  USEMODULE += ztimer
  # native rtc is based on xtimer, no sense in using it and its actually buggy
  ifeq (,$(filter native,$(BOARD)))
    FEATURES_OPTIONAL += periph_rtc
  endif
  include $(RIOTMAKE)/features_check.inc.mk
  ifneq (,$(filter periph_rtc,$(FEATURES_USED)))
    USEMODULE += ztimer_periph_rtc
  else
    FEATURES_OPTIONAL += periph_rtt
    include $(RIOTMAKE)/features_check.inc.mk
    ifneq (,$(filter periph_rtt,$(FEATURES_USED)))
      USEMODULE += ztimer_periph_rtt
    else
      USEMODULE += ztimer_periph_timer
    endif
  endif
endif
```

Reply:
Thanks! Added